examples/generated/test/react-dom/style/VisibilityTest.js

```diff
@@ -0,0 +1,69 @@
+import React from "react"
+import styled, { ThemeProvider } from "styled-components"
+
+import colors from "../colors"
+import textStyles from "../textStyles"
+
+export default class VisibilityTest extends React.Component {
+  render() {
+
+    let Title$visible
+
+    Title$visible = this.props.enabled
+    let theme = {
+      "container": { "normal": {} },
+      "inner": { "normal": {} },
+      "title": { "normal": {} },
+      "view": { "normal": {} }
+    }
+    return (
+      <ThemeProvider theme={theme}>
+        <div style={Object.assign({}, styles.container, {})}>
+          {false && <div style={Object.assign({}, styles.inner, {})} />}
```

Comment:
Is this just the resulting of assigning visibility straight as a lone prop instead of assigning through conditional?

Reply:
yeah. I didn't optimize this case at all since really you should just delete it from the Lona component file... but it'll at least look correct if you don't